dnf-behave-tests/features/plugin-api.feature

```diff
@@ -0,0 +1,35 @@
+Feature: Plugin API
+
+
+Background:
+Given I do not disable plugins
+Given I do not set config file
+  And I create and substitute file "/etc/dnf/dnf.conf" with
```

Comment:
A nitpick (or, a rant, really, why isn't there just one syntax to do it):
```
Given a
Given b
Given c
```
or
```
Given a
  And b
  And c
```

Reply:
updated, thanks